app/integration/project-generator/src/test/resources/io/syndesis/integration/project/generator/testGenerateProject/integration.json

```diff
@@ -53,7 +53,8 @@
               "kind": "ENDPOINT",
               "entrypoint": "direct:extension"
             },
-            "actionType": "step"
+            "actionType": "step",
+            "tags":["expose"]
```

Comment:
What this `expose` will do for the integration?

Reply:
this marks that the integration exposes a service. only when integrations expose some service the project generator will add service related properties (such as camel.component.servlet.header-filter-strategy) to the application.properties. Camel K runtime is more restrictive compared to Spring Boot runtime when it comes to having properties in application.properties that do not match the components used in the integration.